src/dfx/src/util/mod.rs

```diff
@@ -343,6 +349,14 @@ pub fn pretty_thousand_separators(num: String) -> String {
         .collect::<_>()
 }
 
+#[cfg_attr(not(windows), allow(unused))]
```

Comment:
Could we remove this line by referencing `project_dirs` from [here](https://github.com/dfinity/sdk/blob/master/src/dfx/src/config/dfinity.rs#L886)?
```suggestion
```